### PR TITLE
Registration base layout

### PR DESCRIPTION
**Pull request by Ruluk**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -21,10 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <link
-      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
-      rel="stylesheet"
-    />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
```
Comment (dnrgdev) [inline public/index.html:24]: We needs that amount of fonts weight?

File: src/app/shared/components/button/Button.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import './button.scss';
 
 class Button extends Component {
```
Comment (dnrgdev) [inline src/app/shared/components/button/Button.tsx:4]: This component can be a functional component and pass all the props to maintain the button functionalities

File: src/app/shared/components/input/Input.tsx [hunk 3]
```diff
@@ -1,10 +1,8 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './input.scss';
 
-class Input extends Component {
-  render() {
-    return <p className="input">Input</p>;
-  }
-}
+export const Input = ({ type = 'text', placeholder = '' }) => {
```
Comment (dnrgdev) [inline src/app/shared/components/input/Input.tsx:4]: I think you don't need to use an arrow function here if you are not going to use something from a higher scope

File: public/index.html (hunk 1 above)
Comment (yomar-dev) [reply public/index.html:24]: We need to the ask to same question to design team 😂😂😂😂

File: src/app/shared/components/button/Button.tsx (hunk 2 above)
Comment (yomar-dev) [reply src/app/shared/components/button/Button.tsx:4]: You are right.

File: src/app/shared/components/input/Input.tsx (hunk 3 above)
Comment (yomar-dev) [reply src/app/shared/components/input/Input.tsx:4]: Ok, that make sense.

File: src/styles/_button.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+@import 'base/mixins';
+
+%button {
+  @include font-base;
+  padding: $padding-unit 3.4rem;
+  border: none;
+  border-radius: $radius;
+  color: $text-color-white;
+  background-color: $primary-color;
+  font-weight: bold;
```
Comment (farleydiaz) [inline src/styles/_button.scss:11]: please use an environment here

File: src/app/auth/components/createAccountForm/CreateAccountForm.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+  return (
+    <>
+      <RegistrationNav title="Crea la cuenta de tu mascota." />
+
+      <form>
+        <div className="form-control">
+          <label className="form__label">Nombre de tu mascota</label>
+          <input placeholder="ej. Tommy" className="pets-input" />
+        </div>
```
Comment (farleydiaz) [inline src/app/auth/components/createAccountForm/CreateAccountForm.tsx:23]: Maybe you can improve this moving the Inputs to one component in the share folder?

File: src/app/home/components/feed/Feed.tsx [hunk 6]
```diff
@@ -1,8 +1,23 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 class Feed extends Component {
```
Comment (farleydiaz) [inline src/app/home/components/feed/Feed.tsx:4]: The class component it's not necessary yet

**Pull request merged**